src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -33,7 +33,13 @@ const FeedModalItem = (props: FeedEventProps) => {
   const showJaggedEdge = ['claim', 'sendcompleted', 'withdraw', 'receive'].includes(itemType)
 
   return (
-    <ModalWrapper leftBorderColor={mainColor} onClose={buttonPress} showJaggedEdge={showJaggedEdge} fullHeight={true}>
+    <ModalWrapper
+      leftBorderColor={mainColor}
+      itemType={itemType}
+      onClose={buttonPress}
+      showJaggedEdge={showJaggedEdge}
+      fullHeight={true}
```

Comment:
I dont see where fullHeight is being used

Reply:
used in ModalContainer component